src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -167,6 +167,18 @@
 #       --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
 #     factory-reset: true
 #     quiet: true
+#   run17:
```

Comment:
I don't think this will actually do anything - all the tests pass in the CI so you won't hit your code.

Reply:
ah, it's fine - I think the underlying functions are already tested.